website/templates/project/project.mako

```diff
@@ -292,7 +292,7 @@
                   }'></div>
             </div><!-- end containment -->
         % else:
-          <p>No components have been added to this project.</p>
+          <p>No components have been added to this ${node['category'].lower()}.</p>
```

Comment:
fixed in 16d5b32
Now:
![screen shot 2015-05-18 at 3 25 10 pm](https://cloud.githubusercontent.com/assets/4974056/7688404/380a385c-fd72-11e4-8476-6ae61f1a9439.png)